### PR TITLE
[KYUUBI #3445] Flaky Test: Temp view should be deleted after test suite

### PR DESCRIPTION
**Pull request by yikf**
### _Why are the changes needed?_

Fix https://github.com/apache/incubator-kyuubi/issues/3445
temp view should be deleted after test suite, otherwise, some tests which purpose to test show tables will be failure

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/ranger/RangerSparkExtensionSuite.scala [hunk 1]
```diff
@@ -529,21 +529,29 @@ abstract class RangerSparkExtensionSuite extends AnyFunSuite
   test("[KYUUBI #3343] pass temporary view creation") {
     val tempView = "temp_view"
     val globalTempView = "global_temp_view"
-    doAs("denyuser", sql(s"CREATE TEMPORARY VIEW $tempView AS select * from values(1)"))
 
-    doAs(
-      "denyuser",
-      sql(s"CREATE OR REPLACE TEMPORARY VIEW $tempView" +
-        s" AS select * from values(1)"))
+    withCleanTmpResources(Seq((tempView, "view"), (s"global_temp.$globalTempView", "view"))) {
```
Comment (pan3793) [inline extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/ranger/RangerSparkExtensionSuite.scala:533]: Can we refer `org.apache.spark.sql.test.SQLTestUtils`?

**Review by pan3793 — APPROVED** [review]

**Comment by pan3793** [comment]
LGTM, pending CI.

**Comment by pan3793** [comment]
This should go branch-1.6 as well?

**Review by yaooqinn — APPROVED** [review]

**Comment by pan3793** [comment]
Thanks, merging to master
